atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunner.java

```diff
@@ -188,32 +190,67 @@ private SweepResults doRun(TableReference tableRef,
         }
     }
 
+    /**
+     * Returns batches with at least batchConfig.deleteBatchSize blocks per batch.
+     */
     private Iterator<BatchOfCellsToSweep> getBatchesToSweep(Iterator<List<CandidateCellForSweeping>> candidates,
                                                             SweepBatchConfig batchConfig,
                                                             SweepableCellFilter sweepableCellFilter,
                                                             ExaminedCellLimit limit) {
+        Iterator<List<CandidateCellForSweeping>> validCandidates =
```

Comment:
Nothing changed. Thought it would be better to leave it like this, but happy to revert it to the previous implementation.